api/tests/Unit/UserPolicyTest.php

```diff
@@ -0,0 +1,214 @@
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class UserPolicyTest extends TestCase
+{
+    use RefreshDatabase;
+    use WithFaker;
+
+    protected $guest;
+    protected $applicant;
+    protected $otherApplicant;
+    protected $poolOperator;
+    protected $requestResponder;
+    protected $platformAdmin;
+    protected $team;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seed(RolePermissionSeeder::class);
+
+        $this->guest = User::factory()->create([
+            'email' => 'guest-user@test.com',
+            'sub' => 'guest-user@test.com',
+        ]);
+        $this->guest->syncRoles([
+            "guest",
+        ]);
+
+        $this->applicant = User::factory()->create([
+            'email' => 'applicant-user@test.com',
+            'sub' => 'applicant-user@test.com',
+        ]);
+        $this->applicant->syncRoles([
+            "guest",
+            "base_user",
+            "applicant"
+        ]);
+        $this->otherApplicant = User::factory()->create([
+            'email' => 'other-applicant-user@test.com',
+            'sub' => 'other-applicant-user@test.com',
+        ]);
+        $this->otherApplicant->syncRoles([
+            "guest",
+            "base_user",
+            "applicant"
+        ]);
+
+        $this->requestResponder = User::factory()->create([
+            'email' => 'request-responder@test.com',
+            'sub' => 'request-responder@test.com',
+        ]);
+        $this->requestResponder->syncRoles([
+            "guest",
+            "base_user",
+            "applicant",
+            "request_responder"
+        ]);
+
+        $this->poolOperator = User::factory()->create([
+            'email' => 'team-user@test.com',
+            'sub' => 'team-user@test.com',
+        ]);
+        $this->poolOperator->syncRoles([
+            "guest",
+            "base_user",
+            "applicant"
+        ]);
+        $this->team = Team::factory()->create([
+            'name' => 'test-team'
+        ]);
+        $this->poolOperator->attachRole("pool_operator", $this->team);
+
+        $this->platformAdmin = User::factory()->create([
+            'email' => 'admin-user@test.com',
+            'sub' => 'admin-user@test.com',
+        ]);
+        $this->platformAdmin->syncRoles([
+            "guest",
+            "base_user",
+            "applicant",
+            "platform_admin"
+        ]);
+    }
+
+    /**
+     * Only Platform Admins should be able to viewAny
+     *
+     * @return void
+     */
+    public function testViewAny()
+    {
+        $this->assertFalse($this->guest->can('viewAny', User::class));
+        $this->assertFalse($this->applicant->can('viewAny', User::class));
+        $this->assertFalse($this->poolOperator->can('viewAny', User::class));
+        $this->assertFalse($this->requestResponder->can('viewAny', User::class));
+        $this->assertTrue($this->platformAdmin->can('viewAny', User::class));
+    }
```

Comment:
Just popping in to ask, does this work? 😅  I was trying this but it just kept returning `null` every single time 😖

Reply:
Yep, this works! I had to add `use Authorizable;` to the User model.

Reply:
🤦‍♀️ Thanks! That was what I was missing all this time 😝